svgx/darkmode.tsx

```diff
@@ -0,0 +1,16 @@
+export default function DarkMode() {
```

Comment:
There is a components with the same name in chakra-ui to [force dark or light mode](https://chakra-ui.com/docs/styled-system/features/color-mode#forcing-a-specific-mode), so maybe it could be called like NightIcon, DarkModeIcon, (MoonIcon exists in chakra icons as well)